ui/components/privacy-view.jsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+
+export default function PrivacyView() {
+  return (
+    <div className="ml3">
+      <h2 className="h1">Privacy Policy</h2>
+      <h3>PROTECTING PRIVACY AND SECURITY</h3>
+
+      <p className="max-width-3">
+        {/*eslint-disable */}
```

Comment:
Atom was word wrapping but eslint was still barking about line length. I don't love calling it off but didn't want to keep fighting it over long content.